pkg/kubernetes/version.go

```diff
@@ -0,0 +1,197 @@
+package kubernetes
+
+import (
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const (
+	// https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
+	AnnotationSpinnakerArtifactVersion = `artifact.spinnaker.io/version`
+	AnnotationSpinnakerMonikerSequence = `moniker.spinnaker.io/sequence`
+	Annotation
+)
+
+type SpinnakerVersion struct {
+	Long  string
+	Short string
+}
+
+//Create a similar function for labels
+func (c *controller) AddSpinnakerVersionAnnotations(u *unstructured.Unstructured, version SpinnakerVersion) error {
+	annotate(u, AnnotationSpinnakerArtifactVersion, version.Long)
+	annotate(u, AnnotationSpinnakerMonikerSequence, version.Short)
+	var err error
+	gvk := u.GroupVersionKind()
+	if strings.EqualFold(gvk.Kind, "deployment") {
```

Comment:
I think this could be more legible with some newlines.
```go
        annotate(u, AnnotationSpinnakerArtifactVersion, version.Long)
	annotate(u, AnnotationSpinnakerMonikerSequence, version.Short)

	var err error
	gvk := u.GroupVersionKind()

	if strings.EqualFold(gvk.Kind, "deployment") {
```